hail/Makefile

```diff
@@ -234,11 +241,17 @@ pypi-deploy: check-pypi test-dataproc set-docs-sha
 TAG_EXISTS = $(shell git ls-remote --exit-code --tags origin $(HAIL_PIP_VERSION) || echo "does not exist")
 .PHONY: check-tag
 check-tag:
-	if [ -z "$(TAG_EXISTS)" ]; then echo "tag $(HAIL_PIP_VERSION) already exists"; exit 1; fi
+	if [ -z "$(TAG_EXISTS)" ]; then echo "ERROR: tag $(HAIL_PIP_VERSION) already exists"; exit 1; fi
+
+annotation_db_json_url := gs://hail-common/annotationdb/$(HAIL_PIP_VERSION)/annotation_db.json
```

Comment:
i think this should use the SHA version, not the pip version. We've had to upload multiple docs builds for a single pip version in the past.